frontend/src/views/MapExplorerStepsView.vue

```diff
@@ -1,34 +1,33 @@
 <template>
   <div class="iframe-container">
-    <iframe :src="iframeUrl" frameborder="0"></iframe>
+    <iframe ref="iframeRef" :src="iframeUrl" frameborder="0"></iframe>
   </div>
 </template>
 
-<script>
+<script setup>
+import { onMounted, ref, watch } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+const { locale } = useI18n();
+
 const MAPEXPLORER_URL = import.meta.env.VITE_MAPEXPLORER_URL || 'https://stefan-oltmann.de/oni-seed-browser';
+const iframeUrl = MAPEXPLORER_URL
+const iframeRef = ref(null)
+
+watch(locale, () => {
+  if (iframeRef.value && iframeRef.value.contentWindow) {
+    iframeRef.value.contentWindow.postMessage(locale.value, MAPEXPLORER_URL);
+  }
+});
 
-export default {
-  name: 'IframePage',
-  data() {
-    return {
-      iframeUrl: MAPEXPLORER_URL,
-      queryParams: {},
-    }
-  },
-  mounted() {
-    this.queryParams = this.$route.query;
-    this.queryParams.embedded = 'mni'; 
-
-    let url = MAPEXPLORER_URL;
-    if (this.$route.params.seed) {
-      url = `${url}#${this.$route.params.seed}`;
-    }
-    url = `${url}?${new URLSearchParams(this.queryParams).toString()}`;
```

Comment:
This has been deleted but I don't see it re-implemented. By using this if we add ?lang = de, cn, en etc it should tell steps' lib the new language, it also removed the ?embedded=mni query tag too

Reply:
This code essentially passes down any query params from mapsnotincluded to steps map-explorer url

Reply:
I'll add this back in!

Reply:
I've made the changes based on the feedback. Let me know if further adjustments are needed